apps/scene2pset/scene2pset.cc

```diff
@@ -43,8 +44,76 @@ struct AppSettings
     float min_valid_fraction = 0.0f;
     float scale_factor = 2.5f; /* "Radius" of MVS patch (usually 5x5). */
     std::vector<int> ids;
+    bool output_correspondence = false;
 };
 
+struct CorrespondenceViewMetadata
+{
+    unsigned int view_id; 
+    unsigned int width;  // After downsampling. 
```

Comment:
Remove the comments. Either it's clear, or unnecessary, or encode it in the variable name, e.g., `scaled_width`.